modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/resources/META-INF/resources/js/components/WorkflowInstanceTracker.js

```diff
@@ -135,6 +141,15 @@ export default function WorkflowInstanceTracker({workflowInstanceId}) {
 
 	const elements = nodes.concat(transitions);
 
+	useEffect(() => {
+		nodes.map((node) => {
+			if (node.data.current === true) {
```

Comment:
Hi @michaelmdelcavalcanti, `node.data.current` value is always a boolean? If so, could you change this to the following, please?
```suggestion
			if (node.data.current) {
```

Reply:
Hello Aline, fixed.